styx-cli-unshaded/src/test/java/com/spotify/styx/cli/PlainCliOutputTest.java

```diff
@@ -58,6 +58,8 @@
       .schedule(Schedule.DAYS)
       .reverse(false)
       .triggerParameters(TriggerParameters.builder().env("FOO", "bar").build())
+      .created(Instant.parse("2019-01-01T00:00:00Z"))
+      .lastModified(Instant.parse("2019-06-01T00:00:00Z"))
```

Comment:
This should change the expected output as well. So the we need to change the printing behaviour in  `PlainCliOutput` to include two new columns for the new timestamps.